src/main/java/com/google/firebase/internal/DateUtils.java

```diff
@@ -57,9 +58,9 @@
   static final String PATTERN_ASCTIME = "EEE MMM d HH:mm:ss yyyy";
 
   private static final SimpleDateFormat[] DEFAULT_PATTERNS = new SimpleDateFormat[] {
-      new SimpleDateFormat(PATTERN_RFC1123),
-      new SimpleDateFormat(PATTERN_RFC1036),
-      new SimpleDateFormat(PATTERN_ASCTIME)
+      new SimpleDateFormat(PATTERN_RFC1123, Locale.UK),
```

Comment:
Let's follow what http://svn.apache.org/repos/asf/httpcomponents/httpclient/tags/4.3/httpclient/src/main/java/org/apache/http/client/utils/DateUtils.java does and set to `Locale.US`.

Reply:
I fixed it. https://github.com/firebase/firebase-admin-java/pull/327/commits/3e8ec14402afd88a6ad42b89efdbf2bd47d1e6b5